source/geometries/PmtR11410.cc

```diff
@@ -184,44 +194,45 @@ namespace nexus {
     G4double rear_body_irad  = rear_body_diam_/2. - body_thickness_;
 
     front_body_gen_ =
-      new CylinderPointSampler(front_body_irad, front_body_length_, body_thickness_, 0.,
+      new CylinderPointSampler(front_body_irad, front_body_length_,
+                               body_thickness_, 0.,
 			       G4ThreeVector (0., 0., 0.));
 
     medium_body_gen_ =
       new CylinderPointSampler(rear_body_diam_/2., body_thickness_,
 			       front_body_irad - rear_body_diam_/2., 0.,
-			       G4ThreeVector(0., 0., -front_body_length_/2. + body_thickness_/2.));
+			       G4ThreeVector(0., 0., -front_body_length_/2. +
+                                             body_thickness_/2.));
 
     rear_body_gen_ =
-      new CylinderPointSampler(rear_body_irad, rear_body_length_ + body_thickness_,
-			       body_thickness_, 0., G4ThreeVector(0., 0.,
-								  -front_body_length_/2.
-								  - rear_body_length_/2.
-								  + body_thickness_/2.));
+      new CylinderPointSampler(rear_body_irad, rear_body_length_ +
+                               body_thickness_, body_thickness_, 0.,
+                               G4ThreeVector(0., 0., -front_body_length_/2.
+                                             - rear_body_length_/2.
+                                             + body_thickness_/2.));
```

Comment:
Shouldn't these `-` and `+` signs be aligned? They were before.

Reply:
I've fixed it and also corrected the position of the signs.